Makefile

```diff
@@ -1,18 +1,20 @@
-CC=gcc
-CFLAGS=-m32 -O3
-INSTALLDIR=/usr/bin
-BINPATH=bootstrap/mu
+ARCH?=32
```

Comment:
I'm not super familiar with makefiles, but from looking at the docs, IIUC this would allow these variables to be overridden by environment variables, which seems to be discouraged. Also, Muon is all about making parameters explicit so it would make sense to do that for this makefile as well. Can we change these back to just a single equals?

Reply:
That was indeed the intent, to make the values overridable by environment variables. I can change them back if you would like them to be explicit.

Reply:
Yeah I think it'd be best to change them back.